<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panels</class>
 <widget class="QDialog" name="Panels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Укажите параметры панелей</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCmbx">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3000 х 250 мм</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3000 х 100 мм</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2700 х 250 мм</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2700 х 350 мм</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2600 х 240 мм</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Размеры панелей</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество панелей (шт):</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputCnt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="panelsCntOutput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
